worlds/ladx/docs/fr_Links Awakening DX.md

```diff
@@ -0,0 +1,88 @@
+
+S'il n'y a pas de sprite approprié, une lettre sera affichée.
+
+## Que se passe-t-il lorsque le joueur reçoit un objet ?
+
+Lorsque le joueur reçoit un objet, Link le tient au-dessus de sa tête et le montre au monde entier. C'est bon pour les 
+affaires !
+
+## Je ne sais pas quoi faire !
+
+Ce n'est pas une question - mais je vous suggère de cliquer sur l'icône du corbeau sur votre client, ce qui chargera un autotracker compatible AP pour LADXR.
+
+## Sur quoi est basé ce randomiseur ?
+
+Ce randomiseur est basé sur (forké à partir de) le merveilleux travail que daid a fait sur LADXR - https://github.com/daid/LADXR
+
+Le code de l'autotracker pour la communication avec le tracker magpie est directement copié du repo de kbranch - https://github.com/kbranch/Magpie/tree/master/autotracking
+
+### Graphiques
+
+Les images de sprites suivantes ont été incluses avec l'autorisation de leurs auteurs respectifs :
+
+* Par Madam Materia (https://www.twitch.tv/isabelle_zephyr)
+  * Matty_LA
+* Par Linker (https://twitter.com/BenjaminMaksym)
+  * Bowwow
+  * Bunny
+  * Luigi
+  * Mario
+  * Richard
+  * Tarin
+
+Graphisme de l'écran de titre par toomanyteeth✨ (https://instagram.com/toomanyyyteeth)
+
+## Quelques conseils de LADXR...
+
+<h3>Emplacements</h3>
+<p>Les coffres et les clés de donjons sont toujours randomisés. De même, les 3 chansons (Marin, Mambo et Manu) vous donnent un objet si vous leur présentez l'Ocarina. La récompense de 20 seashell du manoir est également aléatoire, mais pas les récompenses de 5 et 10 seashell car elles peuvent être manquées.</p>
+<p>La moblin cave avec Bowwow contient un coffre à la place. Le Color Dungeon donne 2 objets à la fin au lieu d'un choix de tunique. Les autres emplacements d'objets sont : Le toadstool, la récompense pour avoir livré le toadstool, hidden seashells, heart pieces, heart containers, golden leaves, les 3 Mad Batters (améliorations de capacité), la shovel/bow dans la boutique, la tombe du rooster's, et tous les emplacements des clés (tail,slime,angler,face,bird) .</p>
+<p>Enfin, les nouveaux joueurs oublient souvent les endroits suivants : le heart piece caché dans l'eau du  chateau, le heart piece caché dans la bomb cave (écran avant le trade honey), les bonk seashells (courir avec les bottes de pégase contre l'arbre devant Tail Cave, et l'arbre à droite de Mabe Village, à côté de la cabine téléphonique), et l'item du Maître Stalfos dans D5.</p>
+
+<h3>Color Dungeon</h3>
+<p>Le Color Dungeon fait partie du mélange d'objets, et les tuniques rouges/bleues sont mélangées dans le pool d'objets. Cela signifie que la fée à la fin du Color Dungeon donne deux objets aléatoires.</p>
+<p>Pour accéder au Color Dungeon, vous avez besoin du power bracelet, et vous devez pousser les pierres tombales dans le bon ordre : "en bas, à gauche, en haut, à droite, en haut", en allant de la pierre tombale en bas à droite, à celle qui se trouve à sa gauche, au-dessus, puis à droite.</p>
```

Comment:
"power bracelet" is English, I know, apologies for giving up typing in French but French is my second language and I feel my point might get better made in English. "bracelet" is also a French word but "power" is not, leading to in my eyes a strange change of tone when reading out loud, when you have to go from reading an English word straight to a French word.